src/components/Calendar/RangeCalendar/DefaultRangeCalendar/DefaultControlledRangeCalendar.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useRef, useState } from 'react'
+
+import { CalendarProps } from '../../Calendar'
+
+import { DefaultRangeCalendar } from './DefaultRangeCalendar'
+
+export interface DefaultControlledRangeCalendarProps extends CalendarProps {
+  values?: {
+    initialDate: Date
+    finalDate: Date
+  }
+  onChange?(initialDate: Date, finalDate: Date): void
+}
+export const DefaultControlledRangeCalendar = ({
```

Comment:
Use `function` to declare components.

Using `const`, an anonymous (unnamed) function is created and assigned to the const. This different of creating a named function using the `function` keyword.